source/aws-bootstrap-kit/lib/account.ts

```diff
@@ -69,10 +69,21 @@ export interface IAccountProps {
   removalPolicy?: RemovalPolicy;
 }
 
+/**
+ * The type of the AWS account 
+ **/
 export enum AccountType {
+  /**
+   * The account used to deploy CI/CD pipelines (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.CICD) for internal usage).
+   */
   CICD = "CICD",
-  DNS = "DNS",
+  /**
+   * Accounts which will be used to deploy Stage environments (staging/prod ...). (See [here](https://cs.github.com/?scopeName=bk&scope=repo%3Aawslabs%2Faws-bootstrap-kit&q=AccountType.STAGE) for internal usage)
+  **/
   STAGE = "STAGE",
+  /**
+   * Accounts dedicated to developpers work. 
```

Comment:
Aka "sandbox"?

https://aws.amazon.com/blogs/mt/best-practices-creating-managing-sandbox-accounts-aws/

Reply:
indeed SANDBOX works as well , will mention it to the doc